test/conformance/kuttl/webhooks/namespaced-policy-clusterpolicy-wildcard/webhooks.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  labels:
+    webhook.kyverno.io/managed-by: kyverno
+  name: kyverno-resource-validating-webhook-cfg
+webhooks:
+- failurePolicy: Fail
+  rules:
+  - apiGroups:
+    - '*'
+    apiVersions:
+    - '*'
+    operations:
+    - CREATE
+    - UPDATE
+    - DELETE
+    - CONNECT
+    resources:
+    - '*'
+    - pods/ephemeralcontainers
+    scope: '*'
+  - apiGroups:
+      - '*'
+    apiVersions:
+      - '*'
+    operations:
+      - CREATE
+      - UPDATE
+      - DELETE
+      - CONNECT
+    resources:
+      - '*'
+      - pods/ephemeralcontainers
+    scope: 'Namespaced'
```

Comment:
Why both scopes for same resources ?

Reply:
We can refactor it, it was the easiest way to achieve. Otherwise we would need to check for each `Policy` if the same rule is already implemented in `*` scope. 
For the webhook it would not make a difference, I would say.

Reply:
We can always refactor but as we are making changes let's try to get them right as much as possible.